README.md

```diff
@@ -1,87 +1,159 @@
+                                  the default "feat" and "fix". If omitted,
+                                  uses the following list: fix, feat, build,
+                                  chore, ci, docs, perf, refactor, revert,
+                                  style, test, improvement
+
+  -l, --max-subject-length INTEGER
+                                  Maximum characters allowed in the subject of
+                                  the commit message
+
+  -d, --disable TEXT              List of commit message rules to disable. Can
+                                  be one of: C001, C002, C003, C004, C005,
+                                  C006, C007, C008, C009, C010, C011, C012,
+                                  C013, C014, C015, C016, C017, C018, C019,
+                                  C020
+
+  -v, --verbosity LVL             Either CRITICAL, ERROR, WARNING, INFO or
+                                  DEBUG
+
+  --help                          Show this message and exit.
+
+Commands:
+  check     Checks whether commit messages adhere to the Convention Commits...
+  commit    Creates a conventional commit
+  overview  Lists the accepted Conventional Commit tags and Rules (incl.
+```
+
+### Create and Commit a new Conventional Commit
+
+You can use `cm commit` to help you create and commit a Conventional Commit using an interactive wizard:
+
+![example](./resources/commit_example.png)
+
+### Verify commit messages
+
+You can verify single commit messages with the included CLI tool:
 
 ```sh
-$ commisery-verify-msg 8c3349528888a62382afd056eea058843dfb7690
-$ commisery-verify-msg master
-$ commisery-verify-msg :/'refactor'
-$ commisery-verify-msg .git/COMMIT_EDITMSG
-$ commisery-verify-msg my-own-message.txt
+$ cm check 8c3349528888a62382afd056eea058843dfb7690
+$ cm check master
+$ cm check :/'refactor'
+$ cm check .git/COMMIT_EDITMSG
+$ cm check my-own-message.txt
 ```
 
-The exit code of that tool will be non-zero if and only if it found errors in the given commit message.
+Alternatively, it handles commit-ish revision ranges adhering to the `git rev-list` format:
+
+```sh
+$ cm check HEAD~..HEAD
+$ cm check HEAD^2~4 ^HEAD^2~6 ^HEAD^3~2
+$ cm check master..feat/my-feature
+$ cm check HEAD@{yesterday}..HEAD
+$ cm check HEAD@{upstream}..HEAD
+$ cm check :/'refactor'..HEAD
+$ cm check 2fff3d8..8c33495
+```
+
+The exit code of will be non-zero *if and only if* it found errors in the given commit message(s).
+
+> **NOTE**: *in order to remain backwards compatible, the command `commisery-verify-msg` can be used as alternative for `cm check`*
+
+### (Pre-) Commit hook
 
-After that you can use it as a hook in Git to check messages you wrote by creating a `.git/hooks/commit-msg` file with these contents:
+You can use `commisery` as a hook in Git to check messages you wrote by creating a `.git/hooks/commit-msg` file with these contents:
 ```sh
 #!/bin/sh
 exec commisery-verify-msg "$@"
```

Comment:
Why not use here `cm` also consistenyly?

Reply:
Propose to keep the lightweight entry point for commit hook purposes